### DISCUSSION TITLE
Fix for: Error: firebase-tools no longer supports Java version before 11

### DISCUSSION THREAD

**Discussion by rramirez-dev** [mikaelacaron/Basic-Car-Maintenance#317] [General & Questions]
If you get the following error when launching the emulator: 
Error: firebase-tools no longer supports Java version before 11. Please upgrade to Java version 11 or above to continue using the emulators.

Download the latest JDK here: https://www.oracle.com/java/technologies/downloads/#jdk23-mac

**Comment by mikaelacaron** [comment]
Thanks for sharing!

Is there an update we should add to the readme/contributing docs to make this more clear?
@rramirez-dev

**Reply by rramirez-dev** [reply]
We should indicate on contribution read me that the latest firebase tools requires a JDK greater than 11 and we should include the link to the latest JDK because I had difficulty finding it myself.

**Reply by mikaelacaron** [reply]
I'll make an issue for it! want me to assign you to it?

**Reply by rramirez-dev** [reply]
You can assign this issue to me.
